lib/src/signed_video_h26x_auth.c

```diff
@@ -1082,7 +1051,7 @@ has_pending_gop(signed_video_t *self)
     // Collect statistics from pending and hashable NALUs only. The others are either out of date or
     // not part of the validation.
     if (item->validation_status == 'P' && item->nalu && item->nalu->is_hashable) {
-      num_pending_gop_ends += (item->nalu->is_first_nalu_in_gop && !item->need_second_verification);
+      num_pending_gop_ends += (item->nalu->is_first_nalu_in_gop && !item->used_for_linked_hash);
```

Comment:
Under what circumstances can an I-frame be pending and `used_for_linked_hash` at the same time?